cpp/include/data_structure/linked_list/README.md

```diff
@@ -0,0 +1,154 @@
+# Linked Lists
+
+Linked Lists are used as linear data structures which consist of a node. A node holds data as well as a pointer to the next node. Linked Lists easily implement stacks and queues. Linked Lists are used to create graphs and tress. Nodes are not stored at continous memory. Major advantage of linked lists over array sis that the former are dynamic in nature.
+
+### Contents
+
+1. [Singly Linked List](#1-singly-linked-list)
+2. [Doubly Linked List](#2-doubly-linked-list)
+
+---
+
+## 1. Singly Linked List
+
+A linked list is a linear data structure used to maintain a dynamic list of similar elements. The list is maintained so that each element points to the next element and stores some data. To improve performance, the head and tail elements are stored to keep a track of insertions and deletions.
```

Comment:
Since you already describe what a linked list is, I think it would be ideal to point out what's unique about a singly linked list (emphasize the fact that it only points one direction). Same goes for doubly linked list when you add it.